gbpservice/nfp/bin/nfp_configurator.ini

```diff
@@ -17,6 +17,9 @@ modules_dir=gbpservice.nfp.configurator.modules
 reportstate_interval=10
 periodic_interval=9
 
+visibility_vm_ip_address=
+visibility_port=514
+log_level=info
```

Comment:
can we set log_level by default to debug as vyos firewall logs doesn't come in syslog unless log level is debug.? and this log level will be used by vyos only. haproxy doesn't use this.